fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingMutuallyByRecords.java

```diff
@@ -514,6 +536,59 @@ private void infiniteLoopProtection(final Range range, final List<Range> missing
         }
     }
 
+    // anyJumper:
+    //  During the ANY iteration, we wish to avoid cases of two indexers competing on the same fragment.
+    //  It is done by recording the exception and the fragment info, then - during the next iteration:
+    //      If at the same fragment, same range: re-throw
+    //      If at the same fragment, different range: jump to the next fragment
+    boolean anyJumperSaysJump(Range rangeToBuild) {
+        if (anyJumperEx == null || fragmentIterationType != FragmentIterationType.ANY) {
+            return false;
+        }
+        if (LOGGER.isInfoEnabled()) {
+            LOGGER.info(KeyValueLogMessage.build("anyJumper: check if should jump",
```

Comment:
This seems like it should be changed to DEBUG once mutual indexing is a bit less experimental.

Reply:
This only will only log when `anyJumperEx != null`. 
At this point, there are two options: 
1. Jump. In which case, we should probably log something. 
2. Re-throw. In which case, adding an info msg  to a warn one. 
Can we live with it?